docs/auto-creation.md

```diff
@@ -0,0 +1,37 @@
+
+### Automatic Package Creation for Containers
```

Comment:
What do you think about making this just a section of the gateway page rather than its own standalone page?

Reply:
Ok! Sounds good. Moving it.